Makefile

```
@@ -34,3 +34,9 @@ frontend-multistage:
 
 clean:
 	rm aro-hcp-frontend
+
+deploy-frontend:
+	oc process -f ./deploy/aro-hcp-frontend.yml -p ARO_HCP_FRONTEND_IMAGE=${ARO_HCP_FRONTEND_IMAGE} | oc create -f -
```

```suggestion
	oc process -f ./deploy/aro-hcp-frontend.yml -p ARO_HCP_FRONTEND_IMAGE=${ARO_HCP_FRONTEND_IMAGE} | oc replace -f -
```
Nit, suggesting `replace`/`apply` to make it idempotent

---

Updated